plugins/workspace-plugin/src/ca-cert.ts

```diff
@@ -16,9 +16,37 @@ const SS_CRT_PATH = '/tmp/che/secret/ca.crt';
 
 const CA_BUNDLE_PATH = '/tmp/ca-bundle.crt';
 
+/**
+ * Possible locations of default system certificates.
```

Comment:
These are possible locations in general, but not in the case of Che/CRW.
As we run Che on Alpine and CRW on UBI only.

Reply:
What's your proposal?
Remove all except the first and the last?

Reply:
I would like to keep all of those file to be sure it is working.
If you want, I can remove the comments, to make it less confusing.

Reply:
So, simplifying the list and leave only for Alpine and for RHEL/Fedora (as I understand UBI is RHEL)
But in fact, the default certificate has the same location for RHEL and Fedora.

Reply:
BTW, do you remember our discussion about having a common code for getting a certificate?
This will be an improvement in my next PR.